[2022-01-25 06:35:59,985] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-18T00:00:00+00:00 [queued]>
[2022-01-25 06:35:59,995] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-18T00:00:00+00:00 [queued]>
[2022-01-25 06:35:59,995] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:35:59,995] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 06:35:59,995] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:00,007] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-18 00:00:00+00:00
[2022-01-25 06:36:00,011] {standard_task_runner.py:52} INFO - Started process 3890 to run task
[2022-01-25 06:36:00,015] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-18T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp_c5688se', '--error-file', '/tmp/tmpbto3ne69']
[2022-01-25 06:36:00,019] {standard_task_runner.py:77} INFO - Job 8: Subtask first_task
[2022-01-25 06:36:00,086] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-18T00:00:00+00:00 [running]> on host ceab856ace11
[2022-01-25 06:36:00,261] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-18T00:00:00+00:00
[2022-01-25 06:36:00,261] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 06:36:00,262] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 06:36:00,271] {subprocess.py:85} INFO - Output:
[2022-01-25 06:36:00,273] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 06:36:00,273] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 06:36:00,299] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220118T000000, start_date=20220125T063559, end_date=20220125T063600
[2022-01-25 06:36:00,348] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 06:36:00,379] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 07:27:43,206] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-18T00:00:00+00:00 [queued]>
[2022-01-25 07:27:43,223] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-18T00:00:00+00:00 [queued]>
[2022-01-25 07:27:43,223] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:43,223] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 07:27:43,223] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:43,235] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-18 00:00:00+00:00
[2022-01-25 07:27:43,239] {standard_task_runner.py:52} INFO - Started process 350 to run task
[2022-01-25 07:27:43,243] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-18T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpp3xk6_jx', '--error-file', '/tmp/tmpzi5efgxo']
[2022-01-25 07:27:43,248] {standard_task_runner.py:77} INFO - Job 8: Subtask first_task
[2022-01-25 07:27:43,433] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-18T00:00:00+00:00 [running]> on host 943a4c094194
[2022-01-25 07:27:43,485] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-18T00:00:00+00:00
[2022-01-25 07:27:43,486] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 07:27:43,486] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 07:27:43,495] {subprocess.py:85} INFO - Output:
[2022-01-25 07:27:43,497] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 07:27:43,497] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 07:27:43,522] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220118T000000, start_date=20220125T072743, end_date=20220125T072743
[2022-01-25 07:27:43,577] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 07:27:43,606] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
